src/Microsoft.OData.Edm/Schema/Interfaces/IEdmOptionalParameter.cs

```diff
@@ -0,0 +1,21 @@
+//---------------------------------------------------------------------
+// <copyright file="IEdmOptionalParameter.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using Microsoft.OData.Edm.Vocabularies;
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// Represents an optional parameter of an EDM operation.
+    /// </summary>
+    public interface IEdmOptionalParameter : IEdmOperationParameter
+    {
+        /// <summary>
+        /// Gets the default value of this property.
+        /// </summary>
+        string DefaultValueString { get; }

```

Comment:
DefaultValueString seems to be used elsewhere in the API, it should probably match, i.e. no change.

---
In reply to: [123293308](https://github.com/OData/odata.net/pull/828#discussion_r123293308) [](ancestors = 123293308)